startup.krun

```diff
@@ -42,6 +42,7 @@ def patch_runner(vm_def):
         assert "iterations_runner." in vm_def.iterations_runner
         _, suffix = vm_def.iterations_runner.rsplit(".", 1)
         vm_def.iterations_runner = os.path.join(STARTUP_RUNNER_DIR, "startup_runner." + suffix)
+    vm_def.instrument = False
```

Comment:
If PyPy instrumentation is on in `warmup.krun` and we don't turn it off in `startup.krun` then I get a parse error of sorts. So, either we need to decide that we keep this in and all instrumentation is off in the startup runner, or we remove this line and fix the startup runners to work with all the instrumentation we have.